### ISSUE TITLE
Kodi crashing after script start

### ISSUE THREAD

**Issue by sh-hedge** [Giftie/script.cinema.experience#23]
Hi.
Is this project still active? Because I would like to use the script and found two issues.
With any version of XBMC, Kodi, Openelc the Mediacenter restarts without any log entry (even with debuggin on).
I start CinemaExperience and Kodi restarts. But I found a way to prevent this behaviour: playing a youtube trailer before starting CE. It sounds funny, but it wont crash if I started a video once before.

Another issue is, that the home_automation script is not triggerd if I stop (not pause) the movie or quit. I would like to switch my lamps on again if the movie is over...

**Comment by horstepipe** [comment]
Did you download CE from here or from the Kodi repo?
I think you have to download it from here, had the same problem...
The version from repo is 4.0.10, from here it's 3.0.12, but i think it's supposed to be 4.0.12. Maybe just a spelling error, or the old version just works better (at least for me)

**Comment by ghost** [comment]
OK, I'll try it this afternoon. Thx

**Comment by horstepipe** [comment]
omg found our "mistake"
it's really stupid:
in git, you have to press the "branch" button on the upper left side and choose "Gotham" instead of "Master". Then you'll get the real v4.0.13 which works perfect here :D
